cypress/locators/message/index.js

```diff
@@ -0,0 +1,12 @@
+import {
+  MESSAGE_TITLE, MESSAGE_TYPE, MESSAGE_PREVIEW,
+  MESSAGE_CHILDREN,
+  MESSAGE_DISMISS_ICON,
```

Comment:
is it possible to make import in less lines?

Reply:
fixed